python/mxnet/rnn/rnn_cell.py

```diff
@@ -404,7 +409,7 @@ def __call__(self, inputs, states):
                                           name="%sslice"%name)
         in_gate = symbol.Activation(slice_gates[0], act_type="sigmoid",
                                     name='%si'%name)
-        forget_gate = symbol.Activation(slice_gates[1], act_type="sigmoid",
+        forget_gate = symbol.Activation(slice_gates[1] + self._forget_bias, act_type="sigmoid",
```

Comment:
yes that's what I meant.

We plan to keep compatibility where possible. Otherwise LSTMCell should be a super set of FusedRNNCell.